### PR TITLE
Move New Mapping rules and Guidelines from Discord to Website

### PR DESCRIPTION
**Pull request by murpii**

### CODE REVIEW

File: www/rules/index.html [hunk 1]
```diff
@@ -16,33 +16,177 @@
...
+      are not allowed.
+    </li>
+  </ul>
+
+  <h4>🏁 Race maps</h4>
+  <ul>
+    <p>DDNet is focused around DDRace but we host a few race maps.</p>
+    <li>
+      Before posting it on DDNet, ensure that the map doesn't already exist on
+      https://uniqueclan.net/. **
```
Comment (furo321) [inline www/rules/index.html:157]: Would be nice if it was clickable.

File: www/index.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <li><a href="/client/">Client&nbsp;Features</a></li>
     <li><a href="/map/">Map&nbsp;Features</a></li>
     <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
-    <li><a href="/rules/">Mapping&nbsp;Rules</a></li>
+    <li><a href="/rules/">Mapping&nbsp;Rules</a> / <a href="/guidelines/">Guidelines</a></li>
```
Comment (furo321) [inline www/index.php:15]: It seems like the all sub pages define their own menus, so you will need to update those as well. Just do a find and replace.

Comment (murpii) [reply www/index.php:15]: This menu is only visible on the main page.

Comment (murpii) [reply www/index.php:15]: Ah, no nevermind. Will fix

**Comment by murpii** [comment]
Uh, okey no idea what happened there.

**Review by def- — APPROVED** [review]

**Pull request merged**